spec/paperclip/thumbnail_spec.rb

```diff
@@ -481,4 +481,20 @@ def to_s
       end
     end
   end
+
+  context "with a really long file name" do
+    before do
+      image_file = File.new(fixture_file("5k.png"), 'rb')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.